pixl_core/src/core/uploader/_dicomweb.py

```diff
@@ -0,0 +1,121 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Uploader subclass for FTPS uploads."""
+
+from __future__ import annotations
+
+import json
+from typing import Optional
+
+import requests
+from decouple import config  # type: ignore [import-untyped]
+from loguru import logger
+
+from core.uploader.base import Uploader
+
+
+class DicomWebUploader(Uploader):
+    """Upload strategy for a DicomWeb server."""
+
+    def __init__(self, project_slug: str, keyvault_alias: Optional[str]) -> None:
+        """Create instance of parent class"""
+        super().__init__(project_slug, keyvault_alias)
+
+    def _set_config(self) -> None:
+        # Use the Azure KV alias as prefix if it exists, otherwise use the project name
+        az_prefix = self.keyvault_alias
+        az_prefix = az_prefix if az_prefix else self.project_slug
+
+        self.user = self.keyvault.fetch_secret(f"{az_prefix}--dicomweb--username")
+        self.password = self.keyvault.fetch_secret(f"{az_prefix}--dicomweb--password")
+        self.orthanc_url = config("ORTHANC_URL")
```

Comment:
Should this be orthanc anon?
```suggestion
        self.orthanc_url = config("ORTHANC_ANON_URL")
```

Reply:
So as this could is typically run from within `orthanc-anon`, this should point to `http://localhost:8042`, which is the default Orthanc URL as I understand it and which is why I give it a more generic name I think.

Reply:
Fair enough, will then want to update it when Jeremy pulls this into the export api